module/VuFind/src/VuFind/ChannelProvider/Random.php

```diff
@@ -104,7 +104,7 @@ public function __construct(
      */
     public function setOptions(array $options)
     {
-        $this->channelSize = $options['channelSize'] ?? 20;
+        $this->batchSize = $options['batchSize'] ?? 20;
```

Comment:
Another default that's out of sync:


```suggestion
        $this->batchSize = $options['batchSize'] ?? 24;
```